Google Cloud/Google VPC Flow Logs/ingest/parser.yml

```
@@ -0,0 +1,92 @@
+name: google_vpc_flow_logs
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+  - name: parse_jsonPayload
+  - name: parse_labels
+  - name: google_vpc_flow_logs
+  - name: parse_connection
+  - name: parse_src_location
+  - name: parse_dest_pvc
+  - name: parse_dest_instance
+  - name: dest_gke_details
+
+stages:
+  parse_jsonPayload:
+    actions:
+      - set:
+          network.bytes: "{{json_event.message.jsonPayload.bytes_sent}}"
+          google_vpc_flow_logs.jsonPayload.end_time: "{{json_event.message.jsonPayload.end_time}}"
+          network.packets: "{{json_event.message.jsonPayload.packets_sent}}"
+          google_vpc_flow_logs.jsonPayload.reporter: "{{json_event.message.jsonPayload.reporter}}"
+          event.start: "{{json_event.message.jsonPayload.start_time}}"
+        filter: "{{json_event.message.jsonPayload != null}}"
+
+  parse_src_location:
+    actions:
+      - set:
+          source.as.number: "{{json_event.message.jsonPayload.src_location.asn}}"
+          source.geo.continent_name: "{{json_event.message.jsonPayload.src_location.continent}}"
+          source.geo.country_name: "{{json_event.message.jsonPayload.src_location.country}}"
```

It seems to be the ISO 3166 alpha-3 country code 
```suggestion
          source.geo.country_iso: "{{json_event.message.jsonPayload.src_location.country.upper()}}"
```